pages/dkp/kaptain/2.0.0/install/air-gapped-2.2/index.md

```diff
@@ -82,7 +82,9 @@ If you added Kaptain after installing DKP, you must make it available by rerunni
 1.  Download the image bundle file:
 
     - Download `kaptain-air-gapped-2.0.0.tar.gz` that will contain the required artifacts to perform an air-gapped installation.
+    <!-- NEW TAR because of Minio -->
     - (Optional) Download the custom image artifacts `kaptain-air-gapped-2.0.0_cpu.tar.gz` or `kaptain-air-gapped-2.0.0_gpu.tar.gz` based on whether you need CPU or GPU for your workloads.
+    <!-- NEW TAR because of Minio -->
```

Comment:
Something like the following:

```suggestion
  Extract the NOTICES.txt and the image bundle archive before use.
	  
	  ```bash
	  tar -xvf kaptain-air-gapped-2.0.0.tar.gz
```
	  ```